hls_vi/generate_metadata.py

```diff
@@ -152,21 +158,22 @@ def normalize_additional_attributes(container: ElementBase) -> None:
     around the `" + "` and (arbitrarily) using the first value as the value of the
     additional attribute.
     """
-    attrs: List[ElementBase] = container.findall("./AdditionalAttribute", None)
-
-    for attr in attrs:
-        value_element: Optional[ElementBase] = attr.find(".//Value", None)
-        value_text: str = value_element.text if value_element is not None else ""
-
-        if value_element is not None:
-            # Replace the text of the additional attribute with the first value
-            # obtained by splitting the text on " + ".  If the text does not contain
-            # " + ", the text remains the same.  For example, "05.11".split(" + ") is
-            # simply ["05.11"], so taking the first element simply produces "05.11".
-            normalized = value_text.split(" + ", 1)[0].strip()
-            value_element.text = (
-                normalized  # pyright: ignore[reportAttributeAccessIssue]
-            )
+    attr_els: List[ElementBase] = container.findall("./AdditionalAttribute", None)
+
+    for attr_el in attr_els:
+        normalize_additional_attribute(attr_el)
+
+
+def normalize_additional_attribute(attr_el: ElementBase) -> None:
+    values_el: ElementBase = attr_el.find("./Values", None)
+
+    for el in iter(values_el):
+        # Replace the text of the additional attribute value with the first value
+        # obtained by splitting the text on " + ".  If the text does not contain
+        # " + ", the text remains the same.  For example, "05.11".split(" + ") is
+        # simply ["05.11"], so taking the first element simply produces "05.11".
+        # value_el = ET.SubElement(values_el, "Value", attrib=None, nsmap=None)
```

Comment:
Is `value_el = ET.SubElement(values_el, "Value", attrib=None, nsmap=None)` part of the comment or maybe a leftover draft / debugging statement?

Reply:
doh! yep, remnant that should have been removed -- I'll remove it